app/build.gradle.kts

```diff
@@ -24,6 +24,7 @@ plugins {
     id("jacoco")
     id("dagger.hilt.android.plugin")
     id("nowinandroid.spotless")
+    id("com.google.firebase.firebase-perf") version "1.4.1"
```

Comment:
Why are you adding an explicit version here? Can we extract that version as we do with other library/plugin versions?

Reply:
Had never done that before. Added a new plugin for this.